doc/getting-started.md

```diff
@@ -0,0 +1,344 @@
+
+To launch your node, you can choose `dogecoind` to run an headless server or `dogecoin-qt` for a server with a GUI.
+
+In a terminal, run :
+```bash
+# Headless node
+$ dogecoind -printtoconsole
+
+# GUI
+$ dogecoin-qt
+```
+
+*Use `--help` to see all available options for each executables.*
+
+Your node should now be running, starting a *syncing process* by downloading the entire blockchain from other nodes. This operation will take hours, days to complete... but your now part of the dogecoin network !
+
+> **Warning :** The rest of the guide will assume the use of an headless node. The RPC server is not exposed with `dogecoin-qt` until you activate the `-server` option.
+
+## Introduction to JSON-RPC API
+
+Dogecoin Core expose a JSON-RPC interface. With it, you can request information about blocks and transactions, send transactions to the networks, manage your wallet and so on...
+
+To interact with the interface, it relies on HTTP requests. Using urls like `http://user:password@127.0.0.1:22555/` and sending commands through json format is feasible.
+
+Some tools like libraries can be built on top of it. `dogecoin-cli` is the default shell client provided with Dogecoin Core to use this API.
+
+The RPC API documentation is available from [bitcoin](https://developer.bitcoin.org/reference/rpc/).  
+To have an overview of all commands, see also :
+
+```bash
+# List all commands
+$ dogecoin-cli help
+
+# Get help for a specific command
+$ dogecoin-cli help COMMAND
+```
+> **Tips** : Abuse of the help menus is allowed ! They can answer to multiple questions.
+
+### Create a wallet
+
+To receive or recover coins, you will need a wallet address and a private key. The *wallet address* is a shareable information to receive dogecoins from all over the world, while the *private key* is a sensitive information who give the ownership on coins associated to a public address.
+
+By default, you already have a wallet address and a private key available.
+
+List it using `getaddressesbyaccount` :
+
+```bash
+$ dogecoin-cli getaddressesbyaccount ""
+[
+  "DBZc6qeBuo7g8La7L3Lhaf39MsJSgbzpMX"
+]
+```
+
+Use `getnewaddress` to generate a new wallet address :
+```bash
+$ dogecoin-cli getnewaddress
+DC13A8qUgAu5rTWweb44TwXZNQbxDYhyo2
+```
+
+And `dumpprivkey` to find the corresponding private key.
+```bash
+$ dogecoin-cli dumprivkey DC13A8qUgAu5rTWweb44TwXZNQbxDYhyo2
+QSqPGRwxeqv8KNEU85jw128g7rt5ognjqaSvjmeouuFAjJ8Bq3ZR
+```
+
+> **Tips :** Dogecoin wallets addresses mainly start with the letter `D` and private keys with `Q`.
+
+You now have two wallet addresses to share to anyone who want to give you some dogecoins !
+
+### Send a transaction
+
+Dogecoin Core follow the [Unspent Transaction Output (UTXO)](https://en.wikipedia.org/wiki/Unspent_transaction_output) model to manage balances. To own some dogecoins is like having a list of unspent transactions. To spend them, you have to compose a new transaction with received unspent transaction.
```

Comment:
- Dogecoin, not Dogecoin Core.
- Would change "unspent transactions" to "unspent outputs" because that's more concise.